src/request_handler.rs

```diff
@@ -241,14 +244,24 @@ pub(crate) mod tests {
     test(|url, dir| async move {
       std::fs::write(dir.join("www").join("some-test-file.txt"), "contents").unwrap();
       let html = get_html(&url).await;
-      guard_unwrap!(let &[a] = css_select(&html, "a").as_slice());
+      guard_unwrap!(let &[a] = css_select(&html, "a[download]").as_slice());
       let file_url = a.value().attr("href").unwrap();
       let file_url = url.join(file_url).unwrap();
       let file_contents = reqwest::get(file_url).await.unwrap().text().await.unwrap();
       assert_eq!(file_contents, "contents");
     });
   }
 
+  #[test]
+  fn listed_files_can_be_played_in_browser() {
+    test(|url, dir| async move {
+      std::fs::write(dir.join("www").join("some-test-file.txt"), "").unwrap();
+      let html = get_html(&url).await;
+      guard_unwrap!(let &[a] = css_select(&html, "a:not([download])").as_slice());
```

Comment:
And I also learned about the [css not pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:not). So this will ignore all the download links.